packages/api/src/setTrustline/setTrustline.ts

```diff
@@ -1,26 +1,28 @@
 import {
   GEM_WALLET,
-  PaymentResponse,
   RequestSetTrustlineMessage,
-  SetTrustlineRequestPayload
+  SetTrustlineRequestPayload,
+  SetTrustlineResponse
 } from '@gemwallet/constants';
 
 import { sendMessageToContentScript } from '../helpers/extensionMessaging';
 
-export const setTrustline = async (payment: SetTrustlineRequestPayload) => {
+export const setTrustline = async (trustlinePayload: SetTrustlineRequestPayload) => {
```

Comment:
Here you don't add the return type of `setTrustline` like it is done for the other APIs?